distributed-process-tests/src/Control/Distributed/Process/Tests/Tracing.hs

```diff
@@ -19,9 +20,13 @@ import Control.Distributed.Process.Debug
 import Control.Distributed.Process.Management
   ( MxEvent(..)
   )
+import qualified Control.Exception as IO (bracket)
+import Data.List (isPrefixOf, isSuffixOf)
 
 #if ! MIN_VERSION_base(4,6,0)
 import Prelude hiding (catch, log)
+#else
+import Prelude hiding ((<*))
```

Comment:
Latest prelude that exported `catch` in prelude 4.4.1, so I suppose we could drop this CPP thing and leave
only `import Prelude` (in case `log` should not be hidden)